pandas/io/tests/parser/common.py

```diff
@@ -439,6 +439,13 @@ def test_get_chunk_passed_chunksize(self):
         piece = result.get_chunk()
         self.assertEqual(len(piece), 2)
 
+    def test_read_chunksize_generated_index(self):
+        # GH 12185
+        reader = self.read_csv(StringIO(self.data1), chunksize=2)
+        df = self.read_csv(StringIO(self.data1))
```

Comment:
See ``test_read_chunksize`` and ``test_read_chunksize_named`` (hence the "``generated_index``" in the new test's name)... am I missing something?